source/sass/patterns/organisms/site-header.scss

```diff
@@ -89,6 +83,5 @@ $siteHeader-block_end_space: 7px !default;
 
 .site-header__image {
   display: block;
-  @include inline-size(100%);
```

Comment:
Removing the inline size restrictions doesn't appear to have any bad effects, but haven't tested much yet.